src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/apis/EntityAPI.java

```diff
@@ -0,0 +1,82 @@
+            return MethodResult.of(null, "Using players in EntityAPI is disabled in the config. Activate it or ask an admin if they can activate it.");
+        return NBTUtil.toLua(entity.serializeNBT());
+    }
+
+    @LuaFunction(mainThread = true)
+    public final Object getName(IArguments arguments) throws LuaException {
+        Entity entity = EntityUtil.getEntityFromUUID(UUID.fromString(arguments.getString(0)));
+        return entity.getName().getString();
+    }
+
+    @LuaFunction(mainThread = true)
+    public final Object getBoundingBox(IArguments arguments) throws LuaException {
+        if (!APConfig.API_CONFIG.enableGetBoundingBox.get())
+            throw new LuaException("This function is disabled in the config. Activate it or ask an admin if they can activate it.");
+        Entity entity = EntityUtil.getEntityFromUUID(UUID.fromString(arguments.getString(0)));
+        if (!APConfig.API_CONFIG.enablePlayerAccess.get() && entity instanceof Player)
+            return MethodResult.of(null, "Using players in EntityAPI is disabled in the config. Activate it or ask an admin if they can activate it.");
+        return LuaConverter.aabbToObject(entity.getBoundingBox());
+    }
+
+    @LuaFunction(mainThread = true)
+    public final Object getPos(IArguments arguments) throws LuaException {
+        if (!APConfig.API_CONFIG.enableGetPos.get())
+            throw new LuaException("This function is disabled in the config. Activate it or ask an admin if they can activate it.");
+        Entity entity = EntityUtil.getEntityFromUUID(UUID.fromString(arguments.getString(0)));
+        if (!APConfig.API_CONFIG.enablePlayerAccess.get() && entity instanceof Player)
+            return MethodResult.of(null, "Using players in EntityAPI is disabled in the config. Activate it or ask an admin if they can activate it.");
+        return LuaConverter.vec3ToLua(entity.position());
+    }
+
+    /**
+     * Returns entity data depending on the class of the entity. Included data is either not included in the entities
+     * NBT data, or is especially useful so is included for convenience. Because most of this data
+     * doesn't overlap with NBT, it ends up mainly being values that exist only at runtime.
+     */
+    @LuaFunction(mainThread = true)
+    public final Object getData(IArguments arguments) throws LuaException {
+        if (!APConfig.API_CONFIG.enableGetData.get())
+            throw new LuaException("This function is disabled in the config. Activate it or ask an admin if they can activate it.");
+        Entity entity = EntityUtil.getEntityFromUUID(UUID.fromString(arguments.getString(0)));
+        boolean detailed = args.count() > 1 ? args.getBoolean(1) : false;
+        if (!APConfig.API_CONFIG.enablePlayerAccess.get() && entity instanceof Player)
+            throw new LuaException("Using players in EntityAPI is disabled in the config. Activate it or ask an admin if they can activate it.");
```

Comment:
```suggestion
            return MethodResult.of(null, "Using players in EntityAPI is disabled in the config. Activate it or ask an admin if they can activate it.");
```
I'm dumb I was pasting outdated code